.github/workflows/workflow.yml

```diff
@@ -7,91 +7,160 @@ jobs:
-        sudo apt-get install -y pandoc
-        python -m pip install --upgrade pip
-
-    - uses: actions/cache@v1
-      id: cache
-      with:
-        path: ~/.cache/pip
-        key: docs-${{ runner.os }}-pip-${{ hashFiles('requirements/main.txt', 'requirements/dev.txt') }}
-        restore-keys: |
-          docs-${{ runner.os }}-pip-
-
-    - name: Install dependencies
-      run: |
-        pip install -q .
-        pip install -q -r requirements/dev.txt
-
-    - name: Copy examples
-      run: |
-        make --directory ./docs copy-examples
-
-    - name: Generate documentation
-      run: |
-        make --directory ./docs generate
-        
-    - name: Build documentation website
-      run: |
-        make --directory ./docs html
-
-    - uses: actions/upload-artifact@v1
-      with:
-        name: docs
-        path: docs/build/html
+      - name: "1. Clone repository"
+        uses: actions/checkout@v2
 
-  deploy-docs:
+      - name: "2. Set up Python 3.6"
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+
+      - name: "3. Update pip and install pandoc"
+        run: |
+          sudo apt-get install -y pandoc
+          python -m pip install --upgrade pip
+
+      - name: "4. Attach cache for pip"
+        uses: actions/cache@v1
+        id: cache
+        with:
+          path: ~/.cache/pip
+          key: docs-${{ runner.os }}-pip-${{ hashFiles('requirements/main.txt', 'requirements/release.txt') }}
+          restore-keys: |
+            docs-${{ runner.os }}-pip-
+
+      - name: "5. Install main and release dependencies"
+        run: |
+          pip install -q .
+          pip install -q -r requirements/release.txt
+
+      - name: "6. Copy examples"
+        run: |
+          make --directory ./docs copy-examples
+
+      - name: "7. Generate API documentation"
+        run: |
+          make --directory ./docs generate
+
+      - name: "8. Build documentation"
+        run: |
+          make --directory ./docs html
+
+      - name: "9. Store documentation artifact"
+        uses: actions/upload-artifact@v1
+        with:
+          name: docs
+          path: docs/build/html
+
+  release:
     needs: [tests, docs]
     runs-on: ubuntu-latest
     if: contains(github.ref, 'master')
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/download-artifact@v1
+      - name: "1. Clone repository"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+
+      - name: "2. Set up Python 3.6"
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+
+      - name: "3. Update pip and install pandoc"
+        run: |
+          sudo apt-get install -y pandoc
+          python -m pip install --upgrade pip
+
+      - name: "4. Install release dependencies"
+        run: |
+          pip install -q -r requirements/release.txt
+
+      - name: "5. Determine next version"
+        uses: endrjuskr/github-tag-action@master
```

Comment:
This one will be changed to original when https://github.com/anothrNick/github-tag-action/pull/58 will be merged.

Reply:
Your change should be available in the latest release (1.21.0): https://github.com/marketplace/actions/github-tag-bump?version=1.21.0.

Thank you @endrjuskr